extension/src/setup/webview/contract.ts

```diff
@@ -8,3 +8,13 @@ export type SetupData = {
   projectInitialized: boolean
   pythonBinPath: string | undefined
 }
+
+export enum Section {
```

Comment:
Should we make this enum name unique (as well as the one in plot)? Importing after typing only section might be confusing if the wrong `Section` is imported.

Reply:
I actually got hit by this already. I will update after #3452.